### PR TITLE
BONUS: 🔥  Hover Effects

### PR DESCRIPTION
**Pull request by rpeterman-gp**
- update link hovers to not require `::before`
- add hover effect to recommendation links
- add hover effects to grid cards

### To Review

- [ ] Hover over grid cards.
- [ ] Hover over links in text content.
- [ ] Hover over recommendation link images.

> I noticed a few recommendation images that were a little weird when they get a border (eg. transparent PNG, text/logo touching edges, rounded corners in image). May want to find or create better images for those cases.

### CODE REVIEW

File: _sass/_grid.scss [hunk 1]
```diff
@@ -35,13 +35,18 @@
...
 
   &:active,
   &:hover { 
     .card-grid-text {
       opacity: 1;
     }
+
+    .card-image-bg {
+      scale: 1.1;
+    }
```
Comment (brandonhundt) [inline _sass/_grid.scss:49]: I like this!

File: _sass/_type.scss [hunk 2]
```diff
@@ -4,32 +4,32 @@
...
-      left: 0;
-      bottom: 3px;
-      width: 100%;
-      height: 5px;
-      z-index: -1;
-      transition: all .3s ease-in-out;
+
+    padding-inline: 0.25rem;
+
+    transition: background-size 200ms;
```
Comment (brandonhundt) [inline _sass/_type.scss]: Is it possible for the transition to expand the height as opposed to the width of the link? (It also feels a little fast, 300ms feels about right)

File: _sass/_type.scss [hunk 3]
```diff
@@ -125,16 +125,6 @@ h4 {
...
-    text-decoration-color: lighten($orange, 25%);
-    text-decoration-thickness: 5px;
-    -webkit-text-decoration-skip: unset; 
-    text-decoration-skip: unset; 
-    text-decoration-skip-ink: none;
-
-    &::before {
-      display: none;
-    }
+    --link-bg-color: #{lighten($orange, 25%)};
```
Comment (brandonhundt) [inline _sass/_type.scss]: I don't think we need this in the `awards-list` now. It's looking more creme-scicle than the actual orange. Feel free to blow away any special styling for the awards list.

File: _sass/_recommendations.scss [hunk 4]
```diff
@@ -1,14 +1,52 @@
...
+			background-image: conic-gradient(var(--_bg-color), var(--_bg-color) var(--_bg-degrees), transparent var(--_bg-degrees), transparent);
+
+			transition: --_bg-degrees 500ms;
+		}
+
+		&:hover, &:focus-visible {
+			--_bg-degrees: 360deg;
+		}
+	}
+
```
Comment (brandonhundt) [inline _sass/_recommendations.scss:46]: This is fun!

File: _sass/_type.scss (hunk 2 above)
Comment (rpeterman-gp) [reply _sass/_type.scss]: 300ms seemed a bit to slow when switched to animate vertically. Went with 250ms.

File: _sass/_type.scss (hunk 3 above)
Comment (rpeterman-gp) [reply _sass/_type.scss]: Agreed

File: _sass/_type.scss [hunk 5]
```diff
@@ -4,32 +4,32 @@
...
-      background-color: rgba(251, 147, 6, 0.75);
-      position: absolute;
-      left: 0;
-      bottom: 3px;
-      width: 100%;
-      height: 5px;
-      z-index: -1;
-      transition: all .3s ease-in-out;
+
+    padding-inline: 0.25rem;
```
Comment (brandonhundt) [inline _sass/_type.scss]: I'm not sure we need the left and right padding, especially now that we are animating up. It makes links in lists look too much indented.

File: _sass/_type.scss (hunk 2 above)
Comment (brandonhundt) [reply _sass/_type.scss]: I like it!

**Review by brandonhundt — APPROVED** [review]
Great! Once we are done debating the padding on the link, lets :shipit: Thanks @rpeterman-gp

**Pull request merged**